modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

```diff
@@ -146,45 +148,54 @@ $ rosa link user-role <arn> <1>
-<4> Creates the account-wide installer, control plane, worker and support roles and corresponding IAM policies. For more information, see _Account-wide IAM role and policy reference_.
-<5> Creates the cluster-specific Operator IAM roles that permit the ROSA cluster Operators to carry out core OpenShift functionality. For more information, see _Account-wide IAM role and policy reference_.
+<1> Specify the prefix to include in the {cluster-manager} IAM role name. The default is `ManagedOpenShift`.
++
+[IMPORTANT]
+====
+You must specify an account-wide role prefix that is unique across your AWS account, even if you use a custom ARN path for your account roles. The custom paths provide only a logical separation for your IAM roles within your account.
+====
++
+<2> Optional: Specify a permissions boundary Amazon Resource Name (ARN) for the role. For more information, see link:https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html[Permissions boundaries for IAM entities] in the AWS documentation.
+<3> Specify a custom ARN path for your account-wide roles. The path must contain alphanumeric characters only and start and end with `/`, for example `/test/path/dev/`. For more information, see _ARN path customization for IAM roles and policies_.
+<4> Select the role creation mode. You can use `auto` mode to automatically create the account wide roles and policies. In `manual` mode, the `rosa` CLI generates the `aws` commands needed to create the roles and policies. In `manual` mode, the corresponding policy JSON files are also saved to the current directory. `manual` mode enables you to review the details before running the `aws` commands manually.
+<5> Creates the account-wide installer, control plane, worker and support roles and corresponding IAM policies. For more information, see _Account-wide IAM role and policy reference_.
++
+[NOTE]
+====
+In this step, the ROSA CLI also automatically creates the account-wide Operator IAM policies that are used by the cluster-specific Operator policies to permit the ROSA cluster Operators to carry out core OpenShift functionality. For more information, see _Account-wide IAM role and policy reference_.
+====
+--
 .. On the *Accounts and roles* page, click *Refresh ARNs* and verify that the installer, support, worker, and control plane account role ARNs are listed.
++
+If more than one set of account roles are available in your AWS account for your cluster version, a drop-down list of *Installer role* ARNs is provided. Select the ARN for the installer role that you want to use with your cluster. The cluster uses the account-wide roles and policies that relate to the selected installer role.
```

Comment:
are available -> is available (since "set" is singular)

Reply:
Thank you for clarifying this. I was a bit unsure because the reference is to more than one set. Resolved.

Reply:
@EricPonvelle has offered the following alternative:

> If you have more than one set of account roles in your AWS account for your cluster version...

I like it also because it is direct and active. I will apply that to the PR.